CMakeLists.txt

```diff
@@ -199,6 +199,14 @@ if (COMMTHREAD)
   target_compile_definitions(SeisSol-lib PUBLIC USE_COMM_THREAD)
 endif()
 
+if (NUMA_AWARE_PINNING)
+  target_compile_definitions(SeisSol-lib PUBLIC USE_NUMA_AWARE_PINNING)
+  find_package(NUMA REQUIRED)
+
+  target_include_directories(SeisSol-lib SYSTEM PUBLIC ${NUMA_INCLUDE_DIR})
+  target_link_libraries(SeisSol-lib PUBLIC ${NUMA_LIBRARY})
+endif()
+
```

Comment:
I would invert that NO_NUMA_AWARE_PINNING

-> should be the default but people can disable it if they're lazy

Reply:
I've kept that as NUMA_AWARE_PINNING because otherwise this would be the only option that is negated. I've made it the default though.